deploy/e2e/role.yaml

```diff
@@ -104,3 +104,4 @@ rules:
   - patch
   - update
   - watch
+  
```

Comment:
[micro-nit] Can we make github happy here by just restoring this file as it was?